src/library/scala/collection/BitSet.scala

```diff
@@ -140,6 +140,62 @@ trait BitSetOps[+C <: BitSet with BitSetOps[C]]
 
   override def isEmpty: Boolean = 0 until nwords forall (i => word(i) == 0)
 
+  override def max[B >: Int](implicit ord: Ordering[B]): Int =
+    if (Ordering.Int eq ord) {
+      var i = nwords - 1
+      var currentWord = 0L
+      while(i >= 0) {
+        currentWord = word(i)
+        if (currentWord != 0L) {
+          return ((i + 1) * WordLength) - java.lang.Long.numberOfLeadingZeros(currentWord) - 1
+        }
+        i -= 1
+      }
+      throw new UnsupportedOperationException("empty.max")
+    } else if (Ordering.Int.reverse eq ord) {
+      val thisnwords = nwords
```

Comment:
Since this code block is near identical to the one below. perhaps it is worth creating a private inlined method? for readability and maintainability

`@Inline private def smallestInt(empty : String)`
`@Inline private def largestInt(empty : String)`

Reply:
Done!